tex/texlive-bin-extra/Portfile

```diff
@@ -35,6 +36,8 @@ texlive.formats      \
 
 texlive.binaries    a2ping a5toa4 adhocfilelist allcm allec allneeded arara arlatex bibtex8 bibtexu bundledoc checklistings chkdvifont chklref chktex chkweb cllualatex cluttex clxelatex ctan-o-mat ctanbib ctangle ctanify ctanupload ctie ctwill ctwill-refsort ctwill-twinx cweave de-macro depythontex deweb dt2dv dtxgen dv2dt dvi2fax dviasm dvibook dviconcat dvicopy dvidvi dvihp dviinfox dvilj dvilj2p dvilj4 dvilj4l dvilj6 dvipos dvired dviselect dvispc dvitodvi dvitype e2pall findhyph fragmaster git-latexdiff gsftopk installfont-tl ketcindy kpsepath kpsetool kpsewhere kpsexpand lacheck latex-git-log latex-papersize latex2man latex2nemeth latexdef latexfileversion latexindent latexpand listings-ext.sh llmk ltxfileinfo ltximg luajithbtex luajittex make4ht match_parens mflua mflua-nowin mfluajit mfluajit-nowin mkjobtexmf mkocp mkofm optexcount pamphletangler patgen pdfbook2 pdfclose pdfcrop pdflatexpicscale pdfopen pdftex-quiet pdftosrc pdfxup pfarrei pkfix pkfix-helper pooltype ps2frag pslatex purifyeps pythontex rpdfcrop spix srcredact sty2dtx synctex tangle tex4ebook texconfig texconfig-dialog texconfig-sys texcount texdef texdiff texdirflatten texdoc texdoctk texfot texlinks texliveonfly texloganalyser texlogfilter texlogsieve texluajit texluajitc texosquery texosquery-jre5 texosquery-jre8 texplate tie tlcockpit tpic2pdftex twill typeoutfileinfo weave xindex
 
+perl5.branches      5.34
```

Comment:
Doesn't seem to be telling the build which perl to use.

Reply:
I am working on this one.